src/material/tooltip/tooltip.ts

```diff
@@ -1105,6 +1105,9 @@ export class TooltipComponent implements OnDestroy {
     const hideClass = this._hideAnimation;
     tooltip.classList.remove(isVisible ? hideClass : showClass);
     tooltip.classList.add(isVisible ? showClass : hideClass);
+    if (this._isVisible !== isVisible) {
+      this._changeDetectorRef.markForCheck();
```

Comment:
Is this because of the `this._isVisible = isVisible;`? If so, I'd suggest putting the setter just above the `markForCheck` to make that obvious

```
if (this._isVisible !== isVisible) {
  this._isVisible = isVisible;
  this._changeDetectorRef.markForCheck();
}
```

Reply:
👍 Done.